.github/dependabot.yml

```diff
@@ -19,3 +19,30 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
+
+  - package-ecosystem: docker
```

Comment:
After reading the document, 
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#docker
I still don't understand what the dependabot does for the Dockerfile. Please could you help give some more explanation?

Reply:
My understanding

if Dockerfile refer to a tagged image from `https://github.com/dependabot-fixtures/docker-with-source` with following
```
LABEL org.opencontainers.image.source="https://github.com/dependabot-fixtures/docker-with-source"
```
then when a new tag is created at that repo, dependabot PR will be created to update the tag.

Reply:
This might be more relevant.
> Dependabot supports updates for Docker tags that use semver versioning, dates, and build numbers. The Docker tag class is located at: https://github.com/dependabot/dependabot-core/blob/main/docker/lib/dependabot/docker/tag.rb

https://github.com/dependabot/dependabot-core/blob/main/docker/README.md